notNeededPackages.json

```diff
@@ -1008,6 +1008,10 @@
             "libraryName": "convert-hrtime",
             "asOfVersion": "3.0.0"
         },
+        "cookie": {
```

Comment:
This is a pretty straightforward removal. *However*, the stub 1.0.0 package is likely to cause problems even so because of `*` dependents in DT. I'd recommend not adding the entry in this file and just deleting the files. That will not publish or deprecate anything, but will stop future contributions.

Reply:
New versions of npm (in [a change merged in July](https://github.com/npm/npm-pick-manifest/pull/33#top)) skip deprecated versions unless you explicitly specify the version anyway, so people will no longer even see the stub unless they explicitly install `@types/cookie@1`, or if it's manually added to a package.json as `^1.0.0`. So I don't think deprecating placeholder packages has much value anymore.

The compiler will ignore `@types` if the package already ships its own types, and the overhead of 150 lines of .d.ts is a lot less than the other unused code in node_modules.

Reply:
Except that'll mean that all packages on DT that depend on cookie (not sure there are any) will pull in the DT package, unupdated, forever, because nothing tells them to install the real deal.

Reply:
(There is 1: react-cookies)

Reply:
I inlined `SerializeOptions` in the `react-cookies` types so we can properly get rid of `cookie`